resources/main.tf

```diff
@@ -327,3 +322,23 @@ resource "kubernetes_secret" "concourse_basic_auth_credentials" {
     password = "${random_string.basic_auth_password.result}"
   }
 }
+
+resource "helm_release" "concourse_proxy" {
```

Comment:
Rename to simply `concourse` (same with `name` attribute below)

Reply:
added it by mistake, now removed renamed to concourse